src/generator/AutoRest.Java.Azure/Templates/AzurePomTemplate.cshtml

```diff
@@ -0,0 +1,138 @@
+﻿@using System
+@using AutoRest.Java.Azure.Templates
+@using AutoRest.Java.Templates
+@using System.Linq;
+@using AutoRest.Java
+@using AutoRest.Java.Model
+@using AutoRest.Java.Azure.Model
+@inherits AutoRest.Core.Template<AutoRest.Java.Azure.Model.CodeModelJva>
+@{
+    var serviceName = Model.ServiceName;
+}
+<!--
+@Header(" * ").TrimMultilineHeader()
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-parent</artifactId>
+        <version>1.1.3-SNAPSHOT</version>
```

Comment:
You can get the beta annotation version from this version though - 1.x.0-SNAPSHOT can generate @Beta(SinceVersion=V1_X_0) and 1.x.y-SNAPSHOT can generate @Beta(SinceVersion=V1_{X+1}_0)